packages/reactotron-app/App/Foundation/TimelineHeader.js

```diff
@@ -121,6 +130,22 @@ class TimelineHeader extends Component {
           </div>
           <div style={Styles.center}>
             <div style={Styles.title}>{TITLE}</div>
+            <div>
+              <select
+                value={selectedConnectionId}
+                onChange={this.handleSelectedConnectionChange}
```

Comment:
Style this yo.... but before that consider if this is where we actually want it to be.